p5rpc.modloader/Merging/Tbl/P3PTblMerger.cs

```diff
@@ -179,4 +243,16 @@ private async Task<byte[]> PatchAiCalc(byte[] extractedTable, List<string> candi
         var patched = patcher.Apply(patches, TblType.AiCalc, bfs);
```

Comment:
Could just `return patcher.Apply(...)` directly. (Same for your `PatchAny` function, ik this one wasn't made by you)

Reply:
Done.